Content.Client/RCD/RCDMenu.xaml

```diff
@@ -12,21 +12,21 @@
 
     <!-- Entry layer (shows main categories) -->
     <ui:RadialContainer Name="Main" VerticalExpand="True" HorizontalExpand="True" Radius="64" ReserveSpaceForHiddenChildren="False">
-        <ui:RadialMenuTextureButton StyleClasses="RadialMenuButton" SetSize="64 64" ToolTip="{Loc 'rcd-component-walls-and-flooring'}" TargetLayer="WallsAndFlooring" Visible="False">
```

Comment:
ah, now i see what u meant - yea, i removed from all of the stuff that is currently used but style code will remain in case someone would like to use control simple texture buttons that i left as an option :>